adam-core/src/test/scala/org/bdgenomics/adam/util/ADAMFunSuite.scala

```diff
@@ -31,7 +31,9 @@ trait ADAMFunSuite extends SparkFunSuite {
   override val properties: Map[String, String] = Map(
     ("spark.serializer", "org.apache.spark.serializer.KryoSerializer"),
     ("spark.kryo.registrator", "org.bdgenomics.adam.serialization.ADAMKryoRegistrator"),
-    ("spark.kryo.referenceTracking", "true"))
+    ("spark.kryo.referenceTracking", "true"),
+    "spark.kryo.registrationRequired" -> "true"
```

Comment:
Could we have this in tuple form, or convert the other entries to map style? I'd just like all to be consistent.